### PR TITLE
EN-10428 Use existing view metadata

### PR DESCRIPTION
**Pull request by rozap**
* Merge the geo metadata with the view's
  existing metadata

### CODE REVIEW

File: bin/start.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+make
+~/.nvm/nvm.sh use 5.1
+GEO_IMPORT_ENV=dev node lib/index.js
```
Comment (vilterp) [inline bin/start.sh]: 🎉

File: es6-lib/config/config.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ default {
   },
   heapDumpOut: '.',
 
-  emitProgressEvery: 200,
+  emitProgressEvery: 20,
```
Comment (vilterp) [inline es6-lib/config/config.js]: 20ms? seconds? comment or putting unit in var name would be ncie

Comment (rantler) [inline es6-lib/config/config.js]: Varname would be best IMO

Comment (rozap) [inline es6-lib/config/config.js]: good call, considering your guesses were both incorrect :stuck_out_tongue: 

changed to `emitProgressEveryRows` - every N rows it does the thing
(and the thing is debounced - so we don't get more than 1 message every 2 seconds)

**Pull request merged**